mri/reconstruct/linear.py

```diff
@@ -114,13 +116,185 @@ def l2norm(self, shape):
             the L2 norm.
         """
         # Create fake data
-        shape = numpy.asarray(shape)
+        shape = np.asarray(shape)
         shape += shape % 2
-        fake_data = numpy.zeros(shape)
+        fake_data = np.zeros(shape)
         fake_data[tuple(zip(shape // 2))] = 1
 
         # Call mr_transform
         data = self.op(fake_data)
 
         # Compute the L2 norm
-        return numpy.linalg.norm(data)
+        return np.linalg.norm(data)
+
+
+class WaveletUD2(object):
+    """The wavelet undecimated operator using pysap wrapper.
+    """
+    def __init__(self, wavelet_id=24, nb_scale=4,
+                 multichannel=False, n_cpu=1, verbose=0):
```

Comment:
add docs also for verbose, explaining what the different levels are.

Reply:
Well its present, without much significance for now, I could as well remove it.

Reply:
Updated as verbosity level of Parallel. This would be helpful in debug